app/scripts/modules/core/application/modal/pagerDutySelectField.directive.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+let angular = require('angular');
+
+module.exports = angular
+  .module('spinnaker.core.application.pagerDutySelectField.directive', [
+    require('../service/pagerDuty.read.service.js'),
+  ])
+  .directive('pagerDutySelectField', function() {
+    return {
+      restrict: 'E',
+      templateUrl: require('./pagerDutySelectField.directive.html'),
+      controller: 'PagerDutySelectFieldCtrl',
+      controllerAs: 'vm',
+      scope: {},
+      bindToController: {
+        component: '=',
+        field: '@'
```

Comment:
probably don't need to make this configurable, since we're only using it in this one place and the `field` won't change - you can hardcode it in the component.